azra-l/festival-fanatic
# Implement send feature

## Overview:
This PR fixes a bug where clicking the save button did not update on the database. It also contains implementation for the share feature.

## Changes
- New endpoint on backend to email the festival via nodemailer
- New component on the front end for the share form

To test:
- Log in to the app and navigate to a detailed results page
- Click on the `Share` button beside the links to the festival webpage
- Fill out the form and click share
- Check your email

<img width="654" alt="Screen Shot 2022-08-03 at 1 18 26 PM" src="https://user-images.githubusercontent.com/96637812/182705569-2e8c3890-3fd4-43b9-adf8-9f1e75569167.png">

<img width="976" alt="Screen Shot 2022-08-03 at 1 14 17 PM" src="https://user-images.githubusercontent.com/96637812/182705609-cce2f3ad-d4f2-4071-9ad8-21548fae3f46.png">

<img width="976" alt="Screen Shot 2022-08-03 at 10 47 13 AM" src="https://user-images.githubusercontent.com/96637812/182705642-b321cb0d-8e4d-4909-9b33-8d0a3931366c.png">
